inspirehep/dojson/conferences/fields/bd1xx.py

```diff
@@ -38,16 +38,26 @@ def acronym(self, key, value):
     self['closing_date'] = value.get('y')
     self['cnum'] = value.get('g')
     self['place'] = value.get('c')
-    self['subtitle'] = value.get('b')
-    self['title'] = value.get('a')
+    self.setdefault('titles', [])
+    titles = {
```

Comment:
We could, but then again we could say that for place, cnum etc. as well. And that gets quite tedious. We could check that title. subtitle and source and indeed "positive" values and not empty. Say:

```python
if value.get('a'): title['title'] = value.get('a')
```